### PR TITLE
Increase block size

### PR DESCRIPTION
**Pull request by dmarcos**
It needs feedback

### CODE REVIEW

**Comment by dmarcos** [comment]
@chrisplatorres Does this feel better?

**Comment by chrisplatorres** [comment]
Trying it out at https://chrisplatorres.github.io/moonrider fork

**Comment by chrisplatorres** [comment]
Feels like a good size.

File: src/components/beat.js [hunk 1]
```diff
@@ -36,8 +36,8 @@ AFRAME.registerComponent('beat-system', {
   // Will be overridden.
   verticalPositions: {
     bottom: 0.95,
-    middle: 1.35,
-    top: 1.75
+    middle: 1.45,
```
Comment (ngokevin) [inline src/components/beat.js]: These values don't affect, middle/bottom blocks overlap.

Comment (dmarcos) [reply src/components/beat.js]: > 
> 
> I think left and right side blocks feel a tiny bit of a reach for my arm lengths, especially in punch mode (although the distances there need to be adjusted anyways).

I think blocks can be smaller and closer in punch mode. Each mode should define its own parameters.

Comment (ngokevin) [reply src/components/beat.js]: In classic mode, i think it feels a bit of a contortion for the upper right / left notes. also need to adjust for blocks to not overlap

Comment (dmarcos) [reply src/components/beat.js]: Using ratios for blocks height should help bring those closer.

File: src/components/beat.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ AFRAME.registerComponent('beat-system', {
   updateVerticalPositions: function () {
     const MIN_BOTTOM_HEIGHT = 0.4;
     const BOTTOM_HEIGHT = 0.95;
-    const MARGIN = 0.4;
+    const MARGIN = 0.5;
```
Comment (dmarcos) [inline src/components/beat.js]: Increase vertical separation

File: src/components/beat.js (hunk 1 above)
Comment (ngokevin) [reply src/components/beat.js]: we can try ratio, but need to adjust to handle sitting mode too

**Comment by dmarcos** [comment]
Good with this and we can address block height separately.

**Comment by ngokevin** [comment]
I think I prefer to address fully since it overall negatively affects playability for me? Block size increase but things are harder to reach as a result.

**Comment by dmarcos** [comment]
I readjust beat positions and size. I tried both sitting and standing and block distances both horizontal and vertical feel good to me.

**Comment by ngokevin** [comment]
ok i'll try it in a bit

File: src/components/beat.js [hunk 3]
```diff
@@ -47,9 +47,9 @@ AFRAME.registerComponent('beat-system', {
 
   // Will be overridden.
   verticalPositions: {
-    bottom: 0.95,
-    middle: 1.35,
-    top: 1.75
+    bottom: 0.90,
```
Comment (ngokevin) [inline src/components/beat.js]: these values don't actually do anything. in my current branch, i stubbed out this object to reduce future confusion

Comment (dmarcos) [reply src/components/beat.js]: Moved new values to where positions are calculated

**Comment by ngokevin** [comment]
I'm going to play with the values a bit for a couple hours.

The blocks are overlapping. Side-by-side blocks have grown into each other. They have also grown along the depth so consecutive blocks are up against each other, we may need to increase the spacing / speed.

For punch mode, I think the scale needs to go down and move closer + higher. I may try to come up with an easy-to-configurate value where you can adjust the block scales while maintaining separation between blocks so they don't overlap.

**Comment by ngokevin** [comment]
Pushed https://github.com/supermedium/moonrider/commit/93d9627668229a8f0a6508ff8f9524a8c74c88bf with additional adjustments, fixes, positioning debug tools, tweaking of curve speed + walls positioning to reduce overlap, and tailoring for the different modes.

**Pull request closed without merging**